### ISSUE TITLE
Excel 2019: cannot report table row/column headers in protected sheets

### ISSUE THREAD

**Issue by Mister-Kayne** [nvaccess/nvda#18533]
When using NVDA Screen Reader with Excel 2019, setting the option to "Report Column/Row Headings" in a worksheet works as expected when the sheet is unprotected. However, when the sheet is protected (using the key combination ALT+R+P+S and providing a password), NVDA fails to read the column and row headings.
